kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/ControlCliArgumentsTestUtil.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.ctl
+
+import org.apache.kyuubi.KyuubiFunSuite
+
+object ControlCliArgumentsTestUtil extends KyuubiFunSuite {
+
+  /** Check whether the script exits and the given search string is printed. */
+  def testPrematureExit(args: Array[String], searchString: String): Unit = {
```

Comment:
how about removing `ControlCliArgumentsTestUtil` and add `testPrematureExit` into `KyuubiFunSuite`?

Reply:
<img width="736" alt="image" src="https://user-images.githubusercontent.com/21362040/173239973-5c46337b-cc84-4661-92c8-60a28d3fdd0f.png">
seems this method only applies to kyuubi-ctl module

Reply:
create a class named `KyuubiCtlFunSuite` and  extend it?